ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -136,8 +136,16 @@
 
 {{#if (eq @mode "edit")}}
   <form onsubmit={{action "createOrUpdateKey" "edit"}}>
-    <div class="box is-sideless is-fullwidth is-marginless is-paddingless">
-      <MessageError @model={{@modelForData}} @errorMessage={{this.error}} />
+    <div class="box is-sideless is-fullwidth is-marginless padding-top">
```

Comment:
this is the modified warning message in the case the user does not have read permissions on the data endpoint. Rare, and hard to get to in the UI, but we had a test covering it before, and wanted to clean up the messaging.